### ISSUE TITLE
子として追加されたシーンオブジェクトに透明度とvisible設定が反映されない

### ISSUE THREAD

**Issue by niusounds** [ejeinc/Meganekko#7]
``` java
SceneObject parent;
SceneObject child;

...

parent.addChildObject(child);
parent.setOpacity(0.5f);
// parent and child are now opacity 0.5
```

``` java
SceneObject parent;
SceneObject child;

...

parent.setOpacity(0.5f);
parent.addChildObject(child);
// parent's opacity is 0.5 but child's opacity is not 0.5!
```

**Comment by gvrfexp** [comment]
I am interested in the issue. When you get a solution, would you please contribute to https://github.com/gvrfexp? Arigatou!

**Comment by niusounds** [comment]
Thank you for posting. I'm working on this issue.

**Comment by niusounds** [comment]
fixed by 948e4c28abda155d270d244a22a6e1941daf95e6
